app/assets/javascripts/disappear.js

```diff
@@ -24,6 +24,4 @@ $(document).ready(function(){
         }
        });
 });
-$(window).load(function(){
-        $('#rebranding').modal('show');
-    });
```

Comment:
@sethaugustus 
Hey Seth, Is there a reason you had this code in the `diasappear.js` file? It's the same as the code from `rebranding-modal.js`, right?